### DISCUSSION TITLE
怎么直接下载谷歌插件，目前中国不能打开谷歌的服务器，实在不好下载，希望能够直接下载插件，离线安装

### DISCUSSION THREAD

**Discussion by welcome2025** [AutomaApp/automa#354] [Q&A]
怎么直接下载谷歌插件，目前中国不能打开谷歌的服务器，实在不好下载，希望能够直接下载插件，离线安装

**Comment by spbdev** [comment]
从release里面下载zip包，解压缩到硬盘合适位置，然后chrome扩展程序打开开发者模式就可以直接选择解压目录安装了；但是解压目录必须保留不能删，否则下次打开chrome扩展就没了，类似于绿色软件那种感觉。
